docs/running-on-kubernetes.md

```diff
@@ -121,8 +121,8 @@ This URI is the location of the example jar that is already in the Docker image.
 
 If your application's dependencies are all hosted in remote locations like HDFS or HTTP servers, they may be referred to
 by their appropriate remote URIs. Also, application dependencies can be pre-mounted into custom-built Docker images.
-Those dependencies can be added to the classpath by referencing them with `local://` URIs and/or setting the
-`SPARK_EXTRA_CLASSPATH` environment variable in your Dockerfiles. The `local://` scheme is also required when referring to
```

Comment:
Did we remove the ability to put jars somewhere in the image and add them to the classpath at runtime? We had this before with https://github.com/apache/spark/pull/20193. 

cc @mccheah

Reply:
We should confirm that the new init-container-less implementation retains the capability of doing that - maybe through an e2e test.

Reply:
@foxish I can add that capability, but as far as I can tell this is not there now.

Reply:
Should be able to specify dependencies with the URI `local://`. However in order for the image itself to specify additional jars without having to list them in spark-submit, you have to put the jars in the same directory as the rest of the Spark distribution jars. It would be good to have an API or environment variable that can point to additional jars.

Reply:
@mccheah yes putting under /opt/spark/jars is one option because its included by default in the classpath. I will add then SPARK_EXTRA_CLASSPATH back and will let the jars there added to the spark-submit in the container, sounds good?